### ISSUE TITLE
Blockquotes

### ISSUE THREAD

**Issue by jjjclarke** [jjjclarke/markdown-py#1]
A **blockquote** in Markdown is implemented by a > preceding a paragraph.

```
> This is a blockquote.
```

> This is a blockquote.

**Comment by jjjclarke** [comment]
For future reference,

```python
def convert_blockquotes(line):
    if re.match(r'>\s*(.*)', line):
        return f'<blockquote>{match.group(1)}</blockquote>'
    return line
```

did NOT work, and complained that match was undefined.

```python
def convert_blockquotes(line):
    m = re.match(r'>\s*(.*)', line)
    
    if m:
        return f'<blockquote>{m.group(1)}</blockquotes>'
    return line
```

This appeared to work, however.

**Comment by jjjclarke** [comment]
Implemented in [704b648](https://github.com/jjjclarke/markdown-py/commit/704b648428816bfaa5851e2bd60d18d76335faea)
